src/openstudio_lib/HVACSystemsView.cpp

```diff
@@ -606,9 +606,14 @@ MechanicalVentilationView::MechanicalVentilationView() {
   ventilationCalcMethodComboBox = new OSComboBox2();
   ventilationCalcMethodComboBox->setEnabled(false);
   ventilationCalcMethodComboBox->addItem("Zone Sum", "ZoneSum");
-  ventilationCalcMethodComboBox->addItem("Ventilation Rate Procedure", "VentilationRateProcedure");
+  ventilationCalcMethodComboBox->addItem("Standard 62.1 Ventilation Rate Procedure", "Standard62.1VentilationRateProcedure");
+  ventilationCalcMethodComboBox->addItem("Standard 62.1 Ventilation Rate Procedure With Limit", "Standard62.1VentilationRateProcedureWithLimit");
   ventilationCalcMethodComboBox->addItem("Indoor Air Quality Procedure", "IndoorAirQualityProcedure");
-  ventilationCalcMethodComboBox->addItem("Proportional Control", "ProportionalControl");
+  ventilationCalcMethodComboBox->addItem("Indoor Air Quality Procedure Generic Contaminant", "IndoorAirQualityProcedureGenericContaminant");
+  ventilationCalcMethodComboBox->addItem("Indoor Air Quality Procedure Combined", "IndoorAirQualityProcedureCombined");
+  ventilationCalcMethodComboBox->addItem("Proportional Control Bseed On Design Occupancy", "ProportionalControlBasedOnDesignOccupancy");
+  ventilationCalcMethodComboBox->addItem("Proportional Control Bseed On Design OA Rate", "ProportionalControlBasedOnDesignOARate");
```

Comment:
This isn't one of the values that OpenStudio.idd has though.

https://github.com/NREL/OpenStudio/blob/55040fb543628bcbeff73532d488d6ad8ba968ee/resources/model/OpenStudio.idd#L21445-L21453

```
  A5, \field System Outdoor Air Method
       \type choice
       \key ZoneSum
       \key Standard62.1VentilationRateProcedure
       \key Standard62.1VentilationRateProcedureWithLimit
       \key IndoorAirQualityProcedure
       \key ProportionalControl
       \key ProportionalControlBasedOnOccupancySchedule
       \key ProportionalControlBasedOnDesignOccupancy
```

Reply:
I filed https://github.com/NREL/OpenStudio/issues/5060 for it, but we can't drop this PR with options that don't exist in 3.7.0